apps/cloud_composer/CMakeLists.txt

```diff
@@ -45,157 +45,169 @@ PCL_SUBSUBSYS_DEPEND(build "${SUBSYS_NAME}" "${SUBSUBSYS_NAME}" DEPS ${SUBSUBSYS
-    endif()
-
-    include(ComposerTool.cmake REQUIRED)
-
-    #FPFH Tool
-    set(FPFH_DEPS pcl_features pcl_kdtree pcl_filters)
-    set(FPFH_SOURCES tools/fpfh_estimation.cpp)
-    set(FPFH_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/fpfh_estimation.h")
-    define_composer_tool (fpfh_estimation "${FPFH_SOURCES}" "${FPFH_HEADERS}" "${FPFH_DEPS}")
-
-    #Normals Tool
-    set(NORMALS_DEPS pcl_features pcl_kdtree)
-    set(NORMALS_SOURCES tools/normal_estimation.cpp)
-    set(NORMALS_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/normal_estimation.h")
-    define_composer_tool (normal_estimation "${NORMALS_SOURCES}" "${NORMALS_HEADERS}" "${NORMALS_DEPS}")
-
-    #Euclidean Clustering Tool
-    set(EC_DEPS pcl_segmentation pcl_kdtree)
-    set(EC_SOURCES tools/euclidean_clustering.cpp)
-    set(EC_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/euclidean_clustering.h")
-    define_composer_tool (euclidean_clustering "${EC_SOURCES}" "${EC_HEADERS}" "${EC_DEPS}")
-
-    #Statistical Outlier Removal Tool
-    set(SOR_DEPS pcl_filters)
-    set(SOR_SOURCES tools/statistical_outlier_removal.cpp)
-    set(SOR_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/statistical_outlier_removal.h")
-    define_composer_tool (statistical_outlier_removal "${SOR_SOURCES}" "${SOR_HEADERS}" "${SOR_DEPS}")
-
-    #Vox Grid Downsample Tool
-    set(VOXDS_DEPS pcl_filters)
-    set(VOXDS_SOURCES tools/voxel_grid_downsample.cpp)
-    set(VOXDS_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/voxel_grid_downsample.h")
-    define_composer_tool (voxel_grid_downsample "${VOXDS_SOURCES}" "${VOXDS_HEADERS}" "${VOXDS_DEPS}")
-
-     #Organized Segmentation
-    set(OSEG_DEPS pcl_segmentation pcl_kdtree)
-    set(OSEG_SOURCES tools/organized_segmentation.cpp "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/impl/organized_segmentation.hpp")
-    set(OSEG_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/organized_segmentation.h")
-    define_composer_tool (organized_segmentation "${OSEG_SOURCES}" "${OSEG_HEADERS}" "${OSEG_DEPS}")
-
-        #Sanitize Cloud Tool
-    set(SAN_DEPS pcl_filters)
-    set(SAN_SOURCES tools/sanitize_cloud.cpp)
-    set(SAN_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/sanitize_cloud.h")
-    define_composer_tool (sanitize_cloud "${SAN_SOURCES}" "${SAN_HEADERS}" "${SAN_DEPS}")
-
-     #Supervoxels
-    set(VSP_DEPS pcl_octree pcl_segmentation)
-    set(VSP_SOURCES tools/supervoxels.cpp "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/impl/supervoxels.hpp")
-    set(VSP_HEADERS "include/pcl/${SUBSYS_NAME}/${SUBSUBSYS_NAME}/tools/supervoxels.h")
-    define_composer_tool (supervoxels "${VSP_SOURCES}" "${VSP_HEADERS}" "${VSP_DEPS}")
-
-    # Add to the compound apps target
-    list(APPEND PCL_APPS_ALL_TARGETS ${EXE_NAME})
-    set(PCL_APPS_ALL_TARGETS ${PCL_APPS_ALL_TARGETS} PARENT_SCOPE)
+if(NOT build)
```

Comment:
Hard to see, does this modify the lines, or only changes indentation? If latter, please put in a separate commit.

Reply:
This just reduce indentation of following lines. Seems Github does not support viewing diff Side-by-Side (compared to GitLab), so it is indeed a bit hard to see changes here.

Reply:
General question to this: Is this change in general ok (in a separate PR)?
```
if(build)
// Whole remaining CMake code
endif() //endif at end of file
```

```
if(build)
  return()
endif()
// Whole remaining CMake code
```

Reply:
Yes, I'm fine with this, less nesting -> easier to read.